docker/entrypoint-generate.sh

```diff
@@ -7,10 +7,8 @@ chown -R keybase:keybase /mnt
 
 # Run everything else as the keybase user
 sudo -i -u keybase bash << EOF
-source docker/env.sh
+source ./env.sh
```

Comment:
the entrypoint scripts shouldn't have to do this. we should use the `env-file` docker run arg: `docker run -env-file=<filename>`

Reply:
I think this is a totally valid piece of feedback, but is probably a change best separated from this PR, right? This codebase already is setup to copy in the env file rather than provide it via that argument. 

For just a bit of historical context, originally this was a yaml config file for the bot so it made more sense to copy it in. Now that it is environment variables, I totally agree that it would make more sense to pass in as an env-file.

Reply:
i see! ok let's change the env stuff in a separate PR.